res/openvpn-gui-res-it.rc

```diff
@@ -2,6 +2,7 @@
  *  OpenVPN-GUI -- A Windows GUI for OpenVPN.
  *
  *  Copyright (C) 2004 Mathias Sundman <mathias@nilings.se>
+ *                2009 Heiko Hund <heikoh@users.sf.net>
```

Comment:
this is not appropriate -- you neither add nor remove copyright lines unless its your own name. I can fix this though.

Reply:
For me "A Windows GUI for OpenVPN" is a description and not a trademark then I transated it.

Reply:
@selvanair 

The copyright strings are tehs ame as english language.
In previous italian language was missing
"*                2009 Heiko Hund <heikoh@users.sf.net>"